crates/ika-node/src/lib.rs

```diff
@@ -864,23 +864,33 @@ impl IkaNode {
                 previous_epoch_last_system_checkpoint_sequence_number,
             );
 
-        let dwallet_mpc_service_exit_sender = Self::start_dwallet_mpc_service(
+        let (
+            consensus_round_completed_sessions_sender,
+            consensus_round_completed_sessions_receiver,
+        ) = mpsc::unbounded_channel();
```

Comment:
this type of channel can overflow:

Note that the amount of available system memory is an implicit bound to the channel. Using an unbounded channel has the ability of causing the process to run out of memory. In this case, the process will be aborted.

It's unsafe to use, replace with a bounded channel.